utilities/rpp-unittests/HOST_NEW/Tensor_host_pkd3.cpp

```diff
@@ -191,68 +270,97 @@ int main(int argc, char **argv)
     dstDescPtr = &dstDesc;
 
     // Set src/dst layouts in tensor descriptors
-
-    srcDescPtr->layout = RpptLayout::NHWC;
-    if (pln1OutTypeCase)
+    if (layout_switch == 0)
     {
-        strcat(funcType, "_toPLN1");
-        dstDescPtr->layout = RpptLayout::NCHW;
+        srcDescPtr->layout = RpptLayout::NHWC;
+        if (pln1OutTypeCase)
```

Comment:
This condition check if (pln1OutTypeCase) is independent of layout
Please take this outside of if (layout_switch == 0)